acceptance/setup/pre_suite/70_install_released_puppetdb.rb

```diff
@@ -1,8 +1,8 @@
 # We skip this step entirely unless we are running in :upgrade mode.
 if (test_config[:install_mode] == :upgrade)
   step "Install most recent released PuppetDB on the PuppetDB server for upgrade test" do
-    install_puppetdb(database, test_config[:database], 'latest')
+    install_puppetdb(database, test_config[:database], '2.3.5')
```

Comment:
I think if this could be pulling from env variables (like we have for other settings), defaulting back to latest, perhaps this might the patch more flexible for future use.